lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -70,15 +75,73 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
     and what elements are to be removed.
     """
 
-    @enforce_keys [:to_add, :to_remove]
-    defstruct @enforce_keys
-
-    @type element_type_t :: :header | :segment | :partial_segment
+    # TODO: think of a better name
```

Comment:
You can try to get an inspiration from this repo: https://github.com/AirenSoft/OvenMediaEngine/tree/master/src/projects/publishers/llhls

Reply:
They are in favor of using a word `chunk`.